Extensions/PrimitiveDrawing/Extension.cpp

```diff
@@ -99,20 +99,40 @@ void DeclarePrimitiveDrawingExtension(gd::PlatformExtension& extension) {
   obj.AddAction("RoundedRectangle",
                 _("Rounded rectangle"),
                 _("Draw a rounded rectangle on screen"),
-                _("Draw from _PARAM1_;_PARAM2_ to _PARAM3_;_PARAM4_ a rounded rectangle (radius : _PARAM5_) with _PARAM0_"),
+                _("Draw from _PARAM1_;_PARAM2_ to _PARAM3_;_PARAM4_ a rounded rectangle (radius: _PARAM5_) with _PARAM0_"),
                 _("Drawing"),
                 "res/actions/roundedRectangle24.png",
                 "res/actions/roundedRectangle.png")
 
       .AddParameter("object", _("Shape Painter object"), "Drawer")
-      .AddParameter("expression", _("Top left side: X Position"))
-      .AddParameter("expression", _("Top left side : Y Position"))
-      .AddParameter("expression", _("Bottom right side : X Position"))
-      .AddParameter("expression", _("Bottom right side : Y Position"))
+      .AddParameter("expression", _("Top left side: X position"))
+      .AddParameter("expression", _("Top left side: Y position"))
+      .AddParameter("expression", _("Bottom right side: X position"))
+      .AddParameter("expression", _("Bottom right side: Y position"))
       .AddParameter("expression", _("Radius ( in pixels )"))
       .SetFunctionName("DrawRoundedRectangle")
       .SetIncludeFile("PrimitiveDrawing/ShapePainterObject.h");
 
+  obj.AddAction("Star",
+                _("Star"),
+                _("Draw a star on screen"),
+                _("Draw at _PARAM1_;_PARAM2_ an star of _PARAM3_ points and _PARAM4_ of radius (inner radius : _PARAM5_, rotation : _PARAM6_ ) "
+                "with _PARAM0_"),
+                _("Drawing"),
+                "res/actions/star24.png",
+                "res/actions/star.png")
+
+      .AddParameter("object", _("Shape Painter object"), "Drawer")
+      .AddParameter("expression", _("X coordinate of the center of the star"))
+      .AddParameter("expression", _("Y coordinate of the center of the star"))
+      .AddParameter("expression", _("Points of the star"))
+      .AddParameter("expression", _("The radius (in pixels)"))
+      .AddParameter("expression", _("The inner radius (in pixels, half radius by default)"))
+      .SetDefaultValue("1")
```

Comment:
I don't know here, because the[ documentation say ](https://pixijs.download/v4.8.6/docs/PIXI.Graphics.html#drawStar)inner radius is by default half of radius.
In pixi renderer i have added ternary conditions, but i don't know how manage this, in extension.cpp or in pixi renderer.

Reply:
Default value is not supported very well. I would suggest to:
- Write "(in pixels, recommended default value: the radius divided by 2)" so that users know what they should use
- in Pixi renderer, if the value is 0 (which will be what GDevelop will be passing if the expression is not fulfilled), then use the radius/2 instead.

for the 2nd point, I'm following the hypothesis that 0 is not really a valid value (and is what GDevelop is passing for empty expression), so we can detect it.